$OpenBSD: patch-src_analysis_CMakeLists_txt,v 1.4 2015/10/31 15:29:44 landry Exp $
dont call bison to generate the parser, and build the pregenerated file instead
--- src/analysis/CMakeLists.txt.orig	Fri Oct 23 14:10:38 2015
+++ src/analysis/CMakeLists.txt	Fri Oct 30 15:44:07 2015
@@ -35,6 +35,7 @@ SET(QGIS_ANALYSIS_SRCS
   raster/qgsrelief.cpp
   raster/qgsrastercalcnode.cpp
   raster/qgsrastercalculator.cpp
+  raster/qgsrastercalcparser.cpp
   raster/qgsrastermatrix.cpp
   vector/mersenne-twister.cpp
   vector/qgsgeometryanalyzer.cpp
@@ -60,7 +61,7 @@ INCLUDE_DIRECTORIES(BEFORE raster)
 
 ADD_FLEX_FILES_PREFIX(QGIS_ANALYSIS_SRCS raster raster/qgsrastercalclexer.ll)
 
-ADD_BISON_FILES_PREFIX(QGIS_ANALYSIS_SRCS raster raster/qgsrastercalcparser.yy)
+#ADD_BISON_FILES_PREFIX(QGIS_ANALYSIS_SRCS raster raster/qgsrastercalcparser.yy)
 
 IF (PEDANTIC AND MSVC)
   # disable warnings
